internal/server/instance/drivers/driver_qemu.go

```diff
@@ -3384,25 +3384,28 @@ func (d *qemu) generateQemuConfigFile(cpuInfo *cpuTopology, mountInfo *storagePo
 
 	cfg = append(cfg, qemuTablet(&tabletOpts)...)
 
-	// Existing vsock ID from volatile.
-	vsockID, err := d.getVsockID()
-	if err != nil {
-		return "", nil, err
-	}
+	// Windows doesn't support virtio-vsock
```

Comment:
Tiny nit on comment style, we usually start with a capital letter and end with a period, so would be:

```
// Windows doesn't support virtio-vsock.
```

(We've very slowly been converting some comments that don't fit the pattern as we change code near them)